components/pipeline-service/base/rbac/pipelines-as-code/role-binding.yaml

```diff
@@ -0,0 +1,39 @@
+kind: RoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: pipeline-service-admin
+  namespace: pipelines-as-code
+subjects:
+  - kind: Group
+    apiGroup: rbac.authorization.k8s.io
+    name: Pipeline Service
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: admin
+---
+kind: RoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: pipelines-as-code-maintainers
+  namespace: pipelines-as-code
+subjects:
+  - kind: Group
+    apiGroup: rbac.authorization.k8s.io
+    name: Pipeline Service
+  - kind: User
+    apiGroup: rbac.authorization.k8s.io
+    name: sbose78
+  - kind: User
+    apiGroup: rbac.authorization.k8s.io
+    name: jduimovich
+  - kind: User
+    apiGroup: rbac.authorization.k8s.io
+    name: chmouel
+  - kind: User
+    apiGroup: rbac.authorization.k8s.io
+    name: Michkov
```

Comment:
remove Michkov since he is leaving Red Hat

Reply:
removed